### ISSUE TITLE
Stats manquantes dans l'admin

### ISSUE THREAD

**Issue by fjg** [betagouv/mrs#530]
- [ ]  il manque le nombre de demande dans le statut "En cours de liquidation" (en gros afficher le compteur pour chacun des statuts existants)
- [ ] afficher la stat "nombre d'assurés" comme sur /stats
- [ ] renommer bascule en "Nombre d'assurés ayant basculé"
- [ ] ajouter la stat "Nombre de demandes d'assurés ayant basculés" comme sur /stats

**Comment by jpic** [comment]
"afficher la stat "nombre d'assurés" comme sur /stats"

est-ce-que c'est all-time comme sur /stats ou est-ce sur la periode demandée ?

merci

**Comment by fjg** [comment]
l'idée c'est comme toutes les stats de cette page filtrée sur les différents filtres si possible.

**Comment by jpic** [comment]
oui je veux bien presenter la donnees comme les autres stats, reste a definir la regle de calcul exact.

du 10 au 12 est-ce-qu'il faut voir le nombre de demandes validees:

- qui ont ete validees entre ces deux dates
- ou qui ont ete crees entre ces deux dates et qui sont validees

merci

**Comment by jpic** [comment]
ah, mauvaise issue mille excuses :joy:

**Comment by fjg** [comment]
le point 1 comme on le fait pour les autres compteurs (créées, rejetées, ... entre x et y).

**Comment by jpic** [comment]
arf je comprends plus, le nombre d'assures entre x et y se calcule comment exactement ? tous ceux qui ont fait une demande entre x et y ?

je comprends pas pk tu compares avec les autres compteurs, pour moi ce sont des calculs completements differents en terme de specifications techniques

**Comment by jpic** [comment]
En fait, ca n'est pas le meme puisque sur /stats on fait juste un count des personnes, eventuellement avec un filtre sur caisse, donc on a pas ce pb. Donc oui, c'est le meme chiffre quand tu enleves les filtres de dates, ma question porte justement sur le cas ou tu en mets ;)

Du coup, dans le cas ou on filtre sur la periode, le nombre d'abonnes n'est pas le nombre de personnes ayant fait une demande sur cette periode, mais le nombre de personne ayant fait une demande qui a ete traitee durant cette periode ?

Donc un demandeur qui fait une demande avant la periode (la veille par exemple), mais qui a ete validee ou rejettee durant cette periode, va compter.

A l'inverse, un demandeur qui fait une demande durant la periode, mais qui est mise en valide/rejet en dehors de la periode ne compte pas, meme si la demande passe inprogress durant la periode ?

J'ai bien compris qu'il y avait un rapport entre toutes les nouvelles stats **cumulatives**, c'est pk j'ai pas commence a pusher sur une feature et pas l'autre ;)

Merci !

**Comment by fjg** [comment]
c'est bien ce que j'ai précisé oui.

**Comment by jpic** [comment]
Ok mille mercis parce que j'etais parti sur une mauvaise piste !

Pour info, ca n'est pas du tout le calcul qu'on a sur /stats, qui est juste un count :joy:

**Comment by jpic** [comment]
Derniere question: cette demande ne concerne pas le graphique, seulement l'affichage du chiffre de la periode ? Pas de nouveaux points sur le graphique actuel ?

**Comment by fjg** [comment]
yes, si tu fais le 2nd graph cf #455, ce sera sur celui-ci qu'on pourrait le mettre. En l'état la demande c'était d'avoir le chiffre. La courbe c'est du bonus :)

**Comment by jpic** [comment]
Sur l'autre graph c'est les demandes soumises et ici les demandes traitees donc c'est pas exactement pareil. Mais la pour commencer et fermer cette issue tranquille je vais ajouter des calculs en live deja. Si on a besoin de points sur le graphique on se fera ca ;)

**Comment by jpic** [comment]
Je confirme que c'est une grosse piece mine de rien, le commit contiendra ce commentaire:

```
        # do we want to refactor the above into a method here or have a deeper
        # look at django's orm documentation as of version 2.1 maybe there's
        # some "recent" feature that refactors better already ?  ok, it might
        # turn out we have to pay this:
        # https://github.com/yourlabs/crudlfap/issues/87 requesting comments
```

Point de dette technique remontee le 5 Mai ;)

**Comment by jpic** [comment]
Sur /stats c'est un [compte de personnes distintes qui ont une demande sur la caisse selectionnee](https://github.com/betagouv/mrs/blob/master/src/mrs/views.py#L92-L94).

Si je comprends bien pour cloturer cette issue c'est d'ajouter un compte de personnes distintes ayant une demande traitee entre deux dates distintes.

Ou alors on ne tient pas compte du filtre entre periodes distintes, cependant j'ai deja bien avance en ce sens, il ne me reste qu'a refactoriser avec le code existant si possible.

**Comment by fjg** [comment]
On tient compte des filtres comme indiqués (date, caisse, institution).

**Comment by jpic** [comment]
On est bien d'accord qu'on veut tester l'effet de bord du changement de status contre les filtres de requetes ?

Les tests sur les timezones sont dedicaces a @fb86 :joy:

**Comment by jpic** [comment]
Voila il n'y a plus qu'a inclure le facteur de timezone et reficeler les tests 3 4 fois et ca pourrait passer en prod :rocket: 

https://github.com/betagouv/mrs/issues/530

**Comment by jpic** [comment]
Question au sujet de :

- renommer bascule en "Nombre d'assurés ayant basculé"

Le problème c'est que ça ne sera plus cohérent avec la dite stat, qui remonte le nombre de dossier considérés comme des bascules, et non un nombre d'assuré.

Qu'en est-il ?
